src/components/Signup.svelte

```diff
@@ -0,0 +1,119 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import ErrorMessage from "./ErrorMessage.svelte";
+  import {
+    handleEmailBlur,
+    handleFirstNameBlur,
+    handleLastNameBlur,
+    handlePasswordBlur
+  } from "../util/homePageBlur.js";
+  let newUser = {
+    email: "",
+    password: "",
+    firstName: "",
+    lastName: ""
+  };
```

Comment:
Similar to how you made a separate `util` file for different methods for handling inputs, you could create a separate js file for data representations (`newUser`, `user`). I'm guessing you'll have a good # of additional data representations later.